.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,75 @@
+name: UnitTests
```

Comment:
looks that now we can also drop https://github.com/php-coveralls/php-coveralls/blob/master/.travis.yml

Reply:
yes, I will remove `.travis.yml` after I am sure about new configuration working fine (actually it is already :) )

Reply:
Done